doc/ref_model/chapters/chapter04.md

```diff
@@ -266,6 +271,15 @@ Table 4-10 shows performance measurement capabilities.
 <p align="center"><b>Table 4-11:</b> Cloud Infrastructure management Resource Management Measurements</p>
 
 
+<a name="4.1.7"></a>
+### 4.1.7 Other Cloud Infrastructure Capabilities
+
+| Ref | Cloud Infrastructure Capability   | Unit/ Value | Definition/Notes |
+|-----|----------------------|---------|-----------------------|
+| o.cap.001 | CPU Architecture | <value> | Values such as x64, ARM, etc. |
```

Comment:
@pgoyal01 The 'o' reference is not defined. And do we need to introduce it? The capabilities mentioned in 4.1.7 are  required by some workloads and in that sense can by exposed to the workloads.   Can it be renamed  'e.cap'?

Reply:
@karinesevilla I struggled with this and so Thanks for the feedback. I will move them to e.cap and then populate the tables in Section 4.2.

Reply:
@karinesevilla Done. Thanks